less/transferwise.less

```diff
@@ -20,6 +20,25 @@
 @brand-warning-hover: #DF8701;
 //@brand-danger-hover:
 
+
+h1 {
```

Comment:
Unfortunately this is the wrong way to do this, there's a file called variables.less which has variables for these things, can you update there instead?